examples/app-setup-javascript/jest.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
Can we please use preset from test-utils? [https://test-utils.nuxtjs.org/setup#configure-jest]. This way we don't need `.babelrc`.

Reply:
PS: We may simplify by merging e2e to jest.config and only include overrides. Since if the purpose of example is app testing, module testing is not needed anymore (at least for a starter example)